### PR TITLE
Sprint 2/modify redaction validation template email

### PR DESCRIPTION
**Pull request by ayocodingit**
# overview

result: 
image
figma: 
image

cc: @jabardigitalservice/jds-backend 

## Evidence
- title: memperbaiki redaksi email forgot password
- project: Desa Digital
- participants: @ayocodingit @rachadiannovansyah @firmanJS @khihadysucahyo

### CODE REVIEW

**Review by firmanJS — APPROVED** [review]

File: src/lang/locales/id.json [hunk 1]
```diff
@@ -20,6 +21,6 @@
   "auth.rejected": "UNAUTHORIZED",
   "auth.email.forgot.password.success": "Berhasil mengirimkan tautan verifikasi ke {{ email }}",
 
-  "subject.forgot.password": "Password Reset",
+  "subject.forgot.password": "Reset Kata Sandi",
```
Comment (firmanJS) [inline src/lang/locales/id.json:24]: Kata sandi sunandar

Comment (khihadysucahyo) [reply src/lang/locales/id.json:24]: kata.ai/career

**Review by rachadiannovansyah — APPROVED** [review]

File: src/lang/locales/id.json (hunk 1 above)
Comment (ayocodingit) [reply src/lang/locales/id.json:24]: wkwkwk

**Review by khihadysucahyo — APPROVED** [review]
dahlah

File: src/lang/locales/id.json (hunk 1 above)
Comment (sandisunandar99) [reply src/lang/locales/id.json:24]: kata dia ,, wkwk

**Pull request merged**